client/themes/morning.css

```diff
@@ -46,8 +46,22 @@ body {
 	padding: 15px 16px;
 }
 
+/* stylelint-disable */
 #chat .search::-webkit-input-placeholder {
-	color: #99a2b4;
+/* stylelint-enable */
+	color: #99a2b4 !important;
+	opacity: .5;
+}
+
+/* stylelint-disable */
+#chat .search::-moz-placeholder {
+/* stylelint-enable */
```

Comment:
It'd be nice to run postcss over these too, though I guess that's maybe more build tool work than it's worth.

Alternatively, perhaps we could have a different stylelint config for themes?